src/WCSimPrimaryGeneratorAction.cc

```diff
@@ -414,6 +419,70 @@ void WCSimPrimaryGeneratorAction::GeneratePrimaries(G4Event* anEvent)
       particleGun->GeneratePrimaryVertex(anEvent);
     }//old muline format
   }//useMuLineEvt
+  
+  else if(useAmBeEvt){ // Diego Costas (diego.costas.rodriguez@usc.es) 2023
+    // Create the energy distribution object
+    G4SPSEneDistribution* nEnergyDist = new G4SPSEneDistribution();
+    G4SPSRandomGenerator* rGen = new G4SPSRandomGenerator();
+
+    // Set the neutron energy distribution from a file
+    nEnergyDist->SetEnergyDisType("Arb");
+    nEnergyDist->ArbEnergyHistoFile("../data/resampled_nSpectrum.txt");
```

Comment:
I don't think this file is committed? It has to be. Please also
- add it to the install in `CMakeLists.txt`
- modify the path like this, to be consistent with (the default path) of other generators
```suggestion
    nEnergyDist->ArbEnergyHistoFile("data/resampled_nSpectrum.txt");
```

Actually, making a new mac file command in `WCSimPrimaryGeneratorMessenger` defaulting to `data/resampled_nSpectrum.txt` would be the best thing